internal/game/game.go

```diff
@@ -159,5 +171,6 @@ func (g *Game) Run() error {
 			g.Msg = "Sorry, I didn't understand that one! (use 'h' for help)"
 		}
 		debug.Printf("Player is now at (%v). Facing (%c)\n", g.player.p, g.player.o)
+		debug.Printf("Gopher is now at (%v).\n", g.gopher.p)
```

Comment:
I'd roll this into the line above so that the whole debug state is given in a single line. So like:
```
debug.Printf("Player is now at (%v). Facing (%c). Gopher at (%v)\n", g.player.p, g.player.o, g.gopher.p)
```

Reply:
@necrophonic thanks on it! 🙏

Reply:
@necrophonic done and tested. ✅